src/components/connected/cra/Components/BigCountRadioButton.js

```diff
@@ -27,20 +48,33 @@ function BigCountRadioButton({ type, label }) {
           dispatch(craActions.updateNbParticipants(Number(cra?.nbParticipants) - 1));
         }
         break;
+      case 'accompagnement':
+        if (cra?.nbParticipants && cra?.nbParticipants >= cra?.nbParticipantsAccompagnement) {
```

Comment:
Est-ce que `cra?.nbParticipants &&` est nécessaire avec la présence de `?` ?
```suggestion
        if (cra?.nbParticipants >= cra?.nbParticipantsAccompagnement) {
```